src/hooks/staticContent.ts

```diff
@@ -77,10 +78,12 @@ export const useStaticContent = <T>({
     }
 
     try {
-      if (data) {
+      if (!_isEmpty(data?.publicJsonFile?.content)) {
         const json = data?.publicJsonFile?.content;
 
         return parseFromJson ? parseFromJson(json) : json;
+      } else {
+        setError(true);
```

Comment:
this is now always fired for the time, data is not fetched entirely before isEmpty can check the real data.
it needs a check for loading state or that the response has finished with some data.

Reply:
#479